SECURITY.md

```diff
@@ -20,6 +20,7 @@ rapidly and post patches within 14 days if possible.
 
 These vulnerabilities are present in the given versions:
 
+* [CVE-2023-5841](https://takeonme.org/cves/CVE-2023-5841.html) 3.1.0, 3.1.1, 3.1.2, 3.1.3 3.1.4 3.1.5 3.1.6 3.1.7 3.1.8 3.1.9 3.1.10 3.1.11 3.2.0 3.2.1
```

Comment:
Thanks for fixing the omission. For consistency, can you link to the nist.gov website? That page in turn links to the takeonme page. 

```suggestion
* [CVE-2023-5841](https://nvd.nist.gov/vuln/detail/CVE-2023-5841) 3.1.0, 3.1.1, 3.1.2, 3.1.3 3.1.4 3.1.5 3.1.6 3.1.7 3.1.8 3.1.9 3.1.10 3.1.11 3.2.0 3.2.1
```